evalai/utils/challenges.py

```diff
@@ -169,6 +170,15 @@ def display_participated_or_hosted_challenges(is_host=False, is_participant=Fals
         echo(style("\nParticipated Challenges\n", bold=True))
 
         if len(challenges) != 0:
+
+            # Filter out past challenges.
+            ongoing_challenges = []
+            for challenge in challenges:
+                end_date = datetime.strptime(challenge["end_date"], "%Y-%m-%dT%H:%M:%SZ")
+                if (end_date > datetime.now()):
+                    ongoing_challenges.append(challenge)
```

Comment:
Please use the `map-filter` pattern here.
Python has a map function and a filter function. `map` can be used to parse the timestamp and `filter` can be used to filter out unwanted elements.

Reply:
@varunagrawal I changed it, can I make it better?